run/django/test/retry.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##
+# retry.sh
+# Provides utility function commonly needed across Cloud Build pipelines to
+# retry commands on failure.
+#
+# Usage:
+# 1. Retry single command:
+#
+# ./retry.sh "CMD"
+#
+# 2. Retry with check:
+#
+# ./retry.sh "gcloud RESOURCE EXISTS?" "gcloud ACTION"
+#
+#
+# Note:
+#   $# - the number of command-line arguments passed
+#   $? - the exit value of the last command
+##
+
+# Usage: try "cmd1" "cmd2"
+# If first cmd executes successfully then execute second cmd
+runIfSuccessful() {
+  echo "running: $1"
+  $($1 > /dev/null)
+  if [ $? -eq 0 ]; then
+    echo "running: $2"
+    $($2 > /dev/null)
+  else return 1
```

Comment:
@averikitsch Updated line from nodejs-docs-samples version: If running in a two-arg state, if the first command failed, this failure wouldn't be captured by the until loop

Reply:
My original assumption here is if the first command fails, we don't want to retry it since it's most likely a "describe" command. I would assume that "describe" commands wouldn't fail or be flaky like delete or deploy commands. The goals is aimed at capturing the failure of the second command.

Reply:
I was using the script in an "create/describe" format, and without this change 409 errors were falling through. Adding this change would extend it's usage. The script comments describe usage for "action" or "exists/action", so I was using it incorrectly. I will address.